README.md.gotmpl

```diff
@@ -0,0 +1,37 @@
+# Datree Admission Webhook
```

Comment:
why this file was added?

Reply:
This is the file that tells the "helm docs" library how to render the README.md file